src/nested_pandas/series/accessor.py

```diff
@@ -585,17 +606,18 @@ def to_flatten_inner(self, field: str) -> pd.Series:
         >>> from nested_pandas import NestedFrame
         >>> from nested_pandas.datasets import generate_data
         >>> nf = generate_data(5, 2, seed=1).rename(columns={"nested": "inner"})
+        >>> nf["b"] = "b"  # Shorten width of example output
```

Comment:
Funny! 🙏 for the comment. Is that because 'black' formatting interferes with doctests?